MAUI/Radio-Button/Visual-Customization.md

```diff
@@ -123,26 +130,30 @@ The `LineBreakMode` allows you to wrap or truncate the text. The default value o
 {% tabs %}
 {% highlight xaml %}
 
-    <syncfusion:SfRadioButton x:Name="RadioButton" IsChecked="True" WidthRequest="200" LineBreakMode="WordWrap" Text="The LineBreakMode allows you to wrap or truncate the text."></syncfusion:SfRadioButton>
+    <syncfusion:SfRadioButton   x:Name="RadioButton" 
+				IsChecked="True" 
+				WidthRequest="200" 
+				LineBreakMode="WordWrap" 
+				Text="The LineBreakMode allows you to wrap or truncate the text."></syncfusion:SfRadioButton>
```

Comment:
Add this tag in next line </syncfusion:SfRadioButton>

Reply:
Added changes